test/commandHandler.ts

```diff
@@ -12,39 +12,7 @@ const mockTournament = new TournamentMock();
 
 initializeBehaviours("mc!", mockDiscord, mockTournament);
 
-describe("Basic test", function () {
-	it("Help", async function () {
-		await discord.simMessage("mc!help", "help");
-		const response = discord.getResponse("help");
-		expect(response).to.equal(
-			"Emcee's documentation can be found at https://github.com/AlphaKretin/emcee-tournament-bot/wiki."
-		);
-	});
-	it("Ping help", async function () {
-		await discord.simPing("ping");
-		const response = discord.getResponse("ping");
-		expect(response).to.equal(
-			"Emcee's documentation can be found at https://github.com/AlphaKretin/emcee-tournament-bot/wiki."
-		);
-	});
-	it("Argument validation", async function () {
-		await discord.simMessage("mc!create", "error");
-		const response = discord.getResponse("error");
-		expect(response).to.equal("Usage: create name|description");
-	});
-});
 describe("Tournament creation commands", function () {
-	it("Create tournament", async function () {
```

Comment:
i assume the tests deleted here are to be restored as proper unit test files before this PR is finished?

Reply:
They already did